spec/bike_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'bike'
+
+describe Bike do
+  it 'is not broken after we create it' do
+    expect(subject).not_to be_broken
+  end
+
+  it 'can be broken' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.